cloud/azure/variables.tf

```diff
@@ -18,3 +18,99 @@ variable "image_tag_name" {
   description = "Tag for container image"
   default     = "latest"
 }
+
+variable "location_name" {
+  type        = string
+  description = "Name of the location for the resource group"
+  default     = "eastus"
+}
+
+variable "vnet_address_space" {
+  type        = list(string)
+  description = "List of vnet address_spaces"
+  default = [
+    "10.0.0.0/16"
+  ]
+}
+
+variable "subnet_address_prefixes" {
+  type        = list(string)
+  description = "List of the apps subnet address prefixes"
+  default = [
+    "10.0.2.0/24"
```

Comment:
suggestion(non-blocking): do you think it would be useful to add a comment here to make sure that this range doesn't overlap with the range for the postgres subnet address prefixes?